deploy/clowdapp.yaml

```diff
@@ -657,6 +682,9 @@ parameters:
 - description: database secret name
   name: FLOORIST_DB_SECRET_NAME
   value: compliance-db
+- name: FLOORIST_HMS_BUCKET_SECRET_NAME
```

Comment:
Don't forget to set up this variable and the secret in app-interface, prior to merging. Otherwise it would fail on stage and prod, and it would block floorist deployment automation. Thank you!